packages/main/src/RatingIndicator.ts

```diff
@@ -233,37 +225,43 @@ class RatingIndicator extends UI5Element {
 		}
 	}
 
-	_onclick(event) {
-		if (this.disabled || this.readonly) {
+	_onclick(e: MouseEvent) {
+		const target = e.target as HTMLElement | UI5Element;
```

Comment:
UI5Element extends HTMLElement so leaving UI5Element only seems to be okey.

Reply:
Done.